lib/spells/effects/Dispel.h

```diff
@@ -27,8 +27,7 @@ namespace effects
 class Dispel : public UnitEffect
 {
 public:
-	Dispel();
-	virtual ~Dispel();
+	virtual ~Dispel() = default;
```

Comment:
1. isn't destructor already virtual in UnitEffect?
2. I'd remove it completely instead of using `default`

Reply:
@kambala-decapitator will look for it. If it is already virtual, it is better to add override instead.

Reply:
Just remove it. There is not need for virtual default destructors in derived classes.

Reply:
removed